src/Cli/dotnet/commands/InstallingWorkloadCommand.cs

```diff
@@ -86,6 +87,7 @@ internal abstract class InstallingWorkloadCommand : WorkloadCommandBase
             _userProfileDir = creationResult.UserProfileDir;
             _sdkVersion = creationResult.SdkVersion;
             _sdkFeatureBand = new SdkFeatureBand(creationResult.SdkVersion);
+            _workloadRootDir = WorkloadFileBasedInstall.IsUserLocal(_dotnetPath, _sdkFeatureBand.ToString()) ? _userProfileDir : _dotnetPath;
```

Comment:
Looking at the implementation of `WorkloadFileBasedInstall.IsUserLocal`, it has its own logic to calculate the feature band (in `GetUserInstallFilePath`).  We should get rid of that and use `SdkFeatureBand` instead to keep everything consistent.